partiql-tests-data/eval/primitives/path.ion

```diff
@@ -353,412 +433,480 @@ path::[
     }
   },
   {
-    name:"pathWildCardOverScalar",
-    statement:"s[*]",
-    assert:{
-      evalMode:[EvalModeCoerce, EvalModeError],
-      result:EvaluationSuccess,
-      output:$bag::[
-        "hello"
-      ]
-    }
+    name: "pathWildCardOverScalar",
+    statement: "s[*]",
+    assert: [
+      {
+        evalMode: EvalModeCoerce,
+        result: EvaluationSuccess,
+        output: $bag::[
+          "hello"
+        ]
+      },
+      {
+        evalMode: EvalModeError,
+        result: EvaluationFail
```

Comment:
Could you please elaborate on why this failed? 

```
SELECT VALUE v1
FROM s as v1
```

Reply:
From PartiQL Specification:

```
The expression e[∗] reduces to (i.e., is equivalent to):
SELECT VALUE v FROM e AS v
```

And,

> In the following cases the expression in the FROM clause item has the wrong type. Under the type checking option, all of these cases raise an error and the query fails. Under the permissive option, the cases proceed as follows
> -- Section 5.1.1: Mistyping Cases

Therefore,
```
<scalar>[*] = SELECT VALUE v FROM <scalar> AS v --> FROM clause item has the wrong type.
```